doc/implementations.rst

```diff
@@ -0,0 +1,28 @@
+Autocrypt-capable MUAs level 1 implementation status
+=====================================================
+
+Last updated: ``2017/11/03``
+
+================= ======== ======== ======== ======== ======== ======== ======== ======== ========
+ MUA/feature      header   keygen   peer     header   recommen encrypt  setup    setup    uid
+                  parsing           state    inject   dation            message  process  decorat
+================= ======== ======== ======== ======== ======== ======== ======== ======== ========
+ `delta-chat`_    y        y        y        y        y        y        started  y        y
+ `k9`_            y        y [1]_   y        y        y        y        branch   branch   y
+ `enigmail`_      y        y        y        y [2]_   n        y        y        y        n
+ `py-autocrypt`_  y        y        y        y        n        n        n        n        y
+ `notmuch`_       y        n        n        n        n        n        n        n        y
+ `mailpile`_
+================= ======== ======== ======== ======== ======== ======== ======== ======== ========
+
+Leyend: TBD
```

Comment:
s/Leyend/Legend/

If TBD means the same thing as "TODO", then we should use "TODO", so that it's easier to find when searching for other TODO items.